src/app/page.tsx

```diff
@@ -1,16 +1,66 @@
+'use client'
+import {
+  CodeMarkdown,
+  Container,
+  FormGithub,
+  MarkdownPreview,
+  NavBar,
+} from '@/components'
+import { useEffect, useState } from 'react'
+
 export default function Home() {
+  const [markdown, setMarkdown] = useState('')
+  const [showCodeMarkdown, setshowCodeMarkdown] = useState(false)
+  const [isLoading, setIsLoading] = useState(false)
```

Comment:
Se recomienda tipar el useState, solo el tipo de datoa a guardar sería suficiente

```suggestion
  const [isLoading, setIsLoading] = useState<boolean>(false)
```